src/awkward/_v2/contents/recordarray.py

```diff
@@ -191,6 +194,12 @@ def content(self, index_or_key):
             )
         return self._contents[index][: self._length]
 
+    def contentitems(self):
+        pairs = []
+        for i in range(len(self._contents)):
+            pairs.append((self._keys[i], self._contents[i]))
+        return pairs
+
```

Comment:
The test code might be calling this, but let's not reimplement it. In this translation, we'll have the opportunity to cut down on infrequently used methods.

The testing code can be changed from `array.contentitems()` to `zip(array.keys, array.contents)`. (Maybe `keys` is a method, not a property?)